boards/nxp/mimxrt1050_evk/mimxrt1050_evk_mimxrt1052_hyperflash.overlay

```diff
@@ -50,20 +50,20 @@
 				label = "mcuboot";
 				reg = <0x00000000 DT_SIZE_K(128)>;
 			};
-			/* The MCUBoot swap-move algorithm uses the last 14 sectors
+			/* The MCUBoot swap-move algorithm uses the last 11 sectors
```

Comment:
> Just curious, how did we get the new numbers?

Briefly ;)
slot0_size = slot1_size+swap_area_size + sector_size
swap_area_size = (slot0_size/sector_size)*3*program_size rounded up to sector_size.

Reply:
Is is worth documenting that someplace? Not sure where

Reply:
> Is is worth documenting that someplace? Not sure where

Agree. It must be added to the MCUBoot documentation.

Reply:
Can you help submit a PR to MCUBoot with this information. It would be useful when adding support for future SoC's

Reply:
> Can you help submit a PR to MCUBoot with this information. It would be useful when adding support for future SoC's

I have hinted to MCUBoot developers to add this information to MCUBoot documentation. If the slot sizes are wrong, a firmware update may fail.

Reply:
> Can you help submit a PR to MCUBoot with this information. It would be useful when adding support for future SoC's

Added: https://github.com/mcu-tools/mcuboot/pull/2103#event-14875301663